src/Elastic.Apm/Elastic.Apm.csproj

```diff
@@ -8,30 +8,28 @@
   <PropertyGroup>
     <RootNamespace>Elastic.Apm</RootNamespace>
     <AssemblyName>Elastic.Apm</AssemblyName>
-    <AssemblyVersion>1.0.0</AssemblyVersion>
```

Comment:
Same as to other `.csproj` files - please don't change these.

Reply:
If you're referring to the change in use of version properties, this is correct (good info [here](https://andrewlock.net/version-vs-versionsuffix-vs-packageversion-what-do-they-all-mean/)). `AssemblyVersion` defaults to `Version` without any prerelease label. `FileVersion` defaults to `AssemblyVersion`. `InformationalVersion` defaults to `Version` with the prerelease label. The end result is the same, but this is the proper way to do it if you don't require specifically different values for the removed version properties other than the defaults (which isn't the case in this case).

However, I'll ensure the version is aligned with _master_ on the base branch in any case.